In-house patch to augment the Python package's manifest to include the
missing JSON files that make up part of the implementation.
This patch is not suitable for pushing upstream.

--- os-client-config-1.16.0/MANIFEST.in.~1~	2016-02-22 09:29:22.000000000 +0000
+++ os-client-config-1.16.0/MANIFEST.in	2016-08-31 10:52:04.701308581 +0000
@@ -3,4 +3,7 @@ include ChangeLog
 exclude .gitignore
 exclude .gitreview
 
-global-exclude *.pyc
\ No newline at end of file
+global-exclude *.pyc
+
+include os_client_config/*.json
+include os_client_config/vendors/*.json
